zaas-client/src/main/java/org/zowe/apiml/zaasclient/config/ConfigProperties.java

```diff
@@ -24,4 +28,23 @@
     private char[] trustStorePassword;
     private boolean httpOnly;
     private boolean nonStrictVerifySslCertificatesOfServices;
+
+    @Tolerate
+    public ConfigProperties() {
```

Comment:
Did you consider using NoArgs and AllArgs annotations instead of this experimental one?

Reply:
It was my first attempt, but it doesn't work together with @Builder. Just this annotation solved it.